versionist.conf.js

```diff
@@ -16,6 +16,8 @@
 
 'use strict';
 
+const _ = require('lodash');
```

Comment:
There is nothing preventing the file from re-using dependencies from the project it lives in. It might be even desirable in some cases, where the hooks defined here have to make HTTP requests, etc, so I guess its fine.